config/vufind/BlenderMappings.yaml

```diff
@@ -0,0 +1,292 @@
+# Fields in each configuration provide mappings from field names used in Blender.ini
+# to field names used in each backend.
+# By default the field names and behavior match those used with Solr.
+# Any missing mapping means that the backend does not support the field. For facets
+# this means that any filter is ignored for the backend. For searches and sorting
```

Comment:
Is it possible we need a setting to control whether unmapped filters are ignored, or unmapped filters cause the entire backend to be skipped? I could imagine, for example, if we had a checkbox filter for a backend-specific field, when the user applies it, we could just short-circuit all the other backends because we know that none of their records match the filter. Apologies if this is already accounted for in a different way -- just sharing thoughts as I read my way down.